subsys/usb/usb_descriptor.c

```diff
@@ -59,6 +59,23 @@ struct dev_common_descriptor {
 		struct usb_ep_descriptor if1_out_ep;
 	} __packed cdc_acm_cfg;
 #endif
+#ifdef CONFIG_USB_DEVICE_NETWORK_ECM
+	struct usb_cdc_ecm_config {
+		struct usb_association_descriptor iad;
```

Comment:
association descriptor is necessary for composite device only, see above

Reply:
@jfischer-phytec-iot fixed